types.d.ts

```diff
@@ -80,7 +80,8 @@ import {
 	WithStatement,
 	YieldExpression
 } from "estree";
-import { ValidationError, validate as validateFunction } from "schema-utils";
```

Comment:
Is `ValidationError` no longer exposed from the package directly?

Reply:
It is exposed directly https://github.com/webpack/schema-utils/blob/3915bae9a2ce3206dff5f7f3dff717af2ae1d527/src/index.js#L3 ,
Not sure why this changed.

Reply:
Anyway. I'll accept that for now. That seem to be a problem in the typings generation, which may need to be fixed